programs/drift/src/state/oracle.rs

```diff
@@ -164,20 +190,23 @@ pub fn get_oracle_price(
             has_sufficient_number_of_data_points: true,
         }),
         OracleSource::Prelaunch => get_prelaunch_price(price_oracle, clock_slot),
-        OracleSource::PythPull => get_pyth_price(price_oracle, clock_slot, 1, true),
-        OracleSource::Pyth1KPull => get_pyth_price(price_oracle, clock_slot, 1000, true),
-        OracleSource::Pyth1MPull => get_pyth_price(price_oracle, clock_slot, 1000000, true),
+        OracleSource::PythPull => get_pyth_price(price_oracle, clock_slot, 1, oracle_source),
+        OracleSource::Pyth1KPull => get_pyth_price(price_oracle, clock_slot, 1000, oracle_source),
+        OracleSource::Pyth1MPull => {
+            get_pyth_price(price_oracle, clock_slot, 1000000, oracle_source)
+        }
         OracleSource::PythStableCoinPull => {
-            get_pyth_stable_coin_price(price_oracle, clock_slot, true)
+            get_pyth_stable_coin_price(price_oracle, clock_slot, oracle_source)
         }
+        OracleSource::PythLazer => get_pyth_price(price_oracle, clock_slot, 1, oracle_source),
     }
 }
 
 pub fn get_pyth_price(
     price_oracle: &AccountInfo,
     clock_slot: u64,
     multiple: u128,
```

Comment:
nit: multiple is implied from the source, you can probably rm the multiple arg from here and add a multiple() method to the enum